src/Concerns/InteractsWithCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+/** @noinspection PhpUnhandledExceptionInspection */
+
+namespace DefStudio\Telegraph\Concerns;
+
+use DefStudio\Telegraph\Telegraph;
+
+/**
+ * @mixin Telegraph
+ */
+
+trait InteractsWithCommands
```

Comment:
+1 for adding a Concern for `InteractsWithCommands`

would you mind to move here also `registerBotCommands` and `unregisterBotCommands` methods? (they are currently placed inside `HasBotsAndChats` concern)

Reply:
Yeah actually that's what I'm planning to do after this PR. Since you request it I'll do it here instead 👌🏻